node/src/components/block_synchronizer/execution_results_acquisition.rs

```diff
@@ -315,18 +323,24 @@ impl ExecutionResultsAcquisition {
             ) => {
                 debug!("apply_block_execution_results_or_chunk: (Pending, ChunkWithProof)");
                 match apply_chunk(block_hash, checksum, HashMap::new(), chunk, None) {
+                    Ok(ApplyChunkOutcome::HadIt { .. }) => {
+                        error!("cannot have already had chunk if in pending mode");
```

Comment:
Isn't this going to be logged in the caller, thanks to [this impl](https://github.com/casper-network/casper-node/pull/4039/files?diff=split&w=1#diff-9af53ec20015fb55667af0566405bf10e6180dd84cc8f6a7b5cc6e6de44b4c5eR181-R185)?

Reply:
Going up the call stack, it's logged at warn level then debug.  Since this is a true programmer error, I believe it should be logged at error level.